net.crossfireatlas.crossfire.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>net.crossfireatlas.crossfire</id>
+	<name>Crossfire RPG</name>
+	<summary>Roguelike multi-user RPG</summary>
+	<metadata_license>CC-BY-SA-3.0</metadata_license>
+	<project_license>GPL-2.0</project_license>
+	<!-- Description taken from https://crossfire.real-time.com -->
+	<description>
+                <p>An introduction to the game of Crossfire, and links to other sites and what they have to say about the game.</p>
+                <p>In the words of its original author Frank Tore Johansen:</p>
+                <p>"Crossfire is a multi-player graphical arcade and adventure game made for the X Windows systems environment. It has certain flavours from other games, especially gauntlet and the rogue-like games (Nethack, Moria, Angband, and Ragnarok.) Any number of players can move around in their own window, finding and using items and battling monsters. They can choose to cooperate or compete in the same world.''</p>
+                <p>In the years of development that have followed, Crossfire has grown to encompass over 150 monsters, about 3000 maps to explore, an elaborate magic system, over 15 character types, a system of skills, and many, many artifacts and treasures.</p>
+	</description>
+	<developer_name>Multiple authors, see http://crossfire.real-time.com/</developer_name>
```

Comment:
I'd change it to "Crossfire developers".

Reply:
Done.